airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -122,6 +122,9 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
       }
 
       if (workflowState.isDeleted()) {
+        if (workflowState.isRunning()) {
+          reportCancelled(false);
```

Comment:
nit: add a log message here indicating that we are cancelling the job due to deleting the connection